src/index.css

```diff
@@ -96,6 +96,52 @@ body {
   padding: 1rem;
 }
 
+.comment-body {
+  background-color: rgb(237, 237, 237);
```

Comment:
Due to the off white background, it is very hard to read the comments as the text is white. I would look to change this to black, or another dark colour which will contrast the white and ensure they are easily readable.

Reply:
I would also say this is the case with the article itself. The mint coloured background with white text isn't the best contrasting. It may work better with black text or a darker shade of mint to contrast the white.